pkg/bbgo/modify.go

```diff
@@ -0,0 +1,74 @@
+package bbgo
```

Comment:
will be better to put this in the file `interact_modify`

Reply:
done